src/include/daos_srv/vos_types.h

```diff
@@ -240,6 +245,8 @@ enum {
 	VOS_OF_PUNCH_PROPAGATE		= (1 << 14),
 	/** replay punch (underwrite) */
 	VOS_OF_REPLAY_PC		= (1 << 15),
+	/** For detecting DTX uncertainty  */
+	VOS_OF_DETECT_UNCERTAINTY	= (1 << 16),
```

Comment:
Same here, I suppose 'uncertainty' here means 'we don't know if the DTX is committed'? Should we use some other term for avoid confusion?

Reply:
Yes, it means "we do not know whether some DTX is committable or not". Similarly as above, I am not sure which one is better.

Reply:
I will remove this flag in my new patch.